docs/starter-kafka.md

```diff
@@ -0,0 +1,91 @@
+# Starter Kafka
+
+The module `sda-commons-starter-kafka` provides autoconfigured Kafka producer and consumer
+configuration.
+
+Based on:
+
+- `org.springframework.boot:spring-boot-starter`
+- `org.springframework.boot:spring-boot-starter-validation`
+- `org.springframework.kafka:spring-kafka`
+
+##  Configuration
+
+| **Property**                                            | **Description**                                                                                                                   | **Default**        | **Example**                                     | **Env**                                              |
+|---------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------|--------------------|-------------------------------------------------|------------------------------------------------------|
+| `sda.kafka.consumer.retry.initialBackOffInterval` _int_ | The initial backoff of the retry in milli seconds.                                                                                | `1000`             | `1500`                                          | `SDA_KAFKA_CONSUMER_RETRY_INITIAL_BACKOFF_INTERVALL` |
+| `sda.kafka.consumer.retry.maxBackOffInterval` _int_     | The max backoff interval  in milli seconds.                                                                                       | `4000`             | `5000`                                          | `SDA_KAFKA_CONSUMER_RETRY_MAX_BACKOFF_INTERVALL`     |
+| `sda.kafka.consumer.retry.backOffMultiplier` _int_      | The multiplier beginning with the initial backoff.                                                                                | `2`                | `1.5`                                           | `SDA_KAFKA_CONSUMER_RETRY_INITIAL_BACKOFF_INTERVALL` |
```

Comment:
To which types are we referring here? Concerning the example this seems to be rather a decimal.